1816.py

```diff
@@ -0,0 +1,3 @@
+class Solution:
+    def truncateSentence(self, s: str, k: int) -> str:
+        return ' '.join(s.split(" ")[0:k])
```

Comment:
Two redundancies: 

1. No need to specify the `' '` character in `split()` when you split base on empty spaces.
2. If you slice the array starting from the 0 index, you don't need to specify the `0` either--just use `[:k]`.